data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1315 @@ mission "Unfettered Jump Drive Trading"
+			choice
+				`	"Yes, thank you."`
+					goto translate
+				`	"Thank you, but I already understand what I need to."`
+				`	"I have no need of help."`
+			`	She seems a bit offended you do not wish her help, but does not press the matter further and proceeds to move away. Most of the Hai here are with what seems to be families or groups of friends, but you do not see your would-be helper join anyone else as she returns to her previous spot in the shifting audience.`
+			choice
+				`	(Ignore her.)`
+					goto continue
+				`	(Go and speak with her.)`
+			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`
+			choice
+				`	"Yes, please."`
+					goto translate
+				`	"No, but I did not mean to offend you, so I came to excuse myself."`
+			`	"Thank you! No offense is taken. I knew humans were not like they often are depicted here..." she trails off without finishing the thought, instead turning back to the proceedings in the stadium as they continue.`
+				goto continue
+			label translate
+			action
+				log "Minor People" "Merora" "A young adolescent Hai that came to Unfettered space after losing her parents to pirates. She is handling the loss very well, at least on the surface."
+			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, most of the Hai present stand up and speaks, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`
```

Comment:
```suggestion
			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, most of the Hai present stand up, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`
```
Speaking would be chaotic and unnecessary here.